Library/Formula/glibc.rb

```diff
@@ -19,6 +21,9 @@ def install
         "--prefix=#{prefix}",
         "--enable-obsolete-rpc",
         "--without-selinux"] # Fix error: selinux/selinux.h: No such file or directory
+      unless OS.mac?
```

Comment:
Same thing. So, what is the best course of action now? Should I close this pull request and submit a new one? Or were you going to fix it and merge it without my interference?